core/src/main/scala/org/broadinstitute/dsde/rawls/expressions/SlickExpressionParser.scala

```diff
@@ -201,45 +221,47 @@ trait SlickExpressionParser extends JavaTokenParsers {
       workspace <- workspaceQuery.findByIdQuery(context.workspaceContext.workspaceId)
       attribute <- workspaceAttributeQuery if attribute.ownerId === workspace.id && attribute.name === attrName.name && attribute.namespace === attrName.namespace
       nextEntity <- entityQuery if attribute.valueEntityRef === nextEntity.id
-    } yield (rootEntity.name, nextEntity)
+    } yield (rootEntity.name, nextEntity, attribute.listLength.isDefined)
```

Comment:
Part 1 of where the magic happens. Here we check if the first ref is a list member and note it.